app/models/participant_profile.rb

```diff
@@ -35,10 +35,21 @@ class ParticipantProfile < ApplicationRecord
     withdrawn: "withdrawn",
   }, _prefix: "training_status"
 
+  attribute :reason_for_new_cohort, :string
+  enum reason_for_new_cohort: {
+    dqt_mismatch_induction_start_date: "dqt_mismatch_induction_start_date",
+    lead_provider_change: "lead_provider_change",
+    payments_frozen_at_previous_cohort: "payments_frozen_at_previous_cohort",
+    provisional_previous_cohort: "provisional_previous_cohort",
+    registration_mistake: "registration_mistake",
+    unknown: "unknown",
+  }, _prefix: true
+
   # Associations
   belongs_to :participant_identity
   belongs_to :schedule, class_name: "Finance::Schedule"
   belongs_to :teacher_profile, touch: true
+  belongs_to :previous_cohort, class_name: "Cohort", optional: true
```

Comment:
Are we sure this will only ever be one cohort? and do we need to be mindful if they migrate 2021 -> 2024 and then back again (would this be `nil` or 2024 in that scenario?)

Reply:
That's why I suggested to put this field in InductionRecord instead, when we can keep track of those back and forth cohort changes. You suggested that paper_trail could be enough if we put this in ParticipantProfile though.

Reply:
Please see the conversation above with @steventux and @tonyheadford about suggestions where to place this.

Reply:
hmm I wonder again if we are trying to group two types of changes under one, I think flagging a new type of participant which spans multiple cohorts is different to a participant which genuinely changed. Happy to chat through this though with the wider group